temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Print statements are generally not used in production code for logging purposes. Consider using a logging framework instead.
```
import logging

logging.basicConfig(level=logging.INFO)
logging.info('hello')
```